### ISSUE TITLE
[Question] Registrar add_to_organization

### ISSUE THREAD

**Issue by danforbes** [substrate-developer-hub/substrate-enterprise-sample#63]
This part of the add_to_organization function looks unclear to me
https://github.com/substrate-developer-hub/substrate-enterprise-sample/blob/19f3b1c7b3ff08f6f0fda252015cb6dc1dad078f/chain/pallets/registrar/src/lib.rs#L91-L98

Can someone help me understand the design? Based on the names, I would it expect it to read something like this:

```rust
// Accounts that belong to a certain organization.
let mut members = Self::members_of(&who);

// Validate organization and account should not be part.
if !members.contains(&account) {
	members.push(account.clone());
	MembersOf::<T>::insert(&who, members);
} else {
```

**Comment by stiiifff** [comment]
Looks weird indeed. I'd have to run through the end-to-end test scenario to know for sure.
A few tests would have definitely helped clear any confusion.

**Comment by danforbes** [comment]
Closed with #64
